R/merge.R

```diff
@@ -9,6 +12,11 @@
         stop("'f' must have the same number of ",dim.type," as 'x'",
             call. = FALSE)
     }
+    # This is done otherwise we lose NA values
+    if ( !na.rm && any(is.na(f)) ) {
+      f <- as.character(f)
+      f[is.na(f)] <- "NA"
```

Comment:
Indentation should be multiplication of 4 spaces. Run BiocCheck::BiocCheck()

(Also if sentence is not in similar format than others, not big deal but we should follow similar style)